spec/requests/api/v1/account/account_index_spec.rb

```diff
@@ -0,0 +1,50 @@
+RSpec.describe "GET '/api/v1/account/listings" do
+  let!(:landlord) { create(:user)}
+  let!(:subscriber) { create(:user)}
+
+  let!(:listing_1) { create(:listing, lead: 'The first lead', scene: 'indoor', landlord_id: landlord.id) }
```

Comment:
please refactor this background

`let!(:landlord_listings) { 2.times { create(:listing, landlord_id: landlord.id) } }`

and the same for the "other listings"